src/test/java/org/jenkinsci/plugins/ssegateway/EventDispatcherTest.java

```diff
@@ -0,0 +1,70 @@
+package org.jenkinsci.plugins.ssegateway;
+
+import static org.junit.Assert.*;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.Method;
+import java.util.Map;
+import java.util.Queue;
+import java.util.concurrent.TimeUnit;
+
+import org.jenkinsci.plugins.pubsub.Message;
+import org.jenkinsci.plugins.pubsub.SimpleMessage;
+import org.jenkinsci.plugins.ssegateway.sse.EventDispatcher;
+import org.junit.BeforeClass;
+import org.junit.Test;
+
+public class EventDispatcherTest {
+
+    private static Method addToRetryQueue;
+    private static Field retryQueue;
+    private static Field subscribers;
+
+    @BeforeClass
+    public static void setup() throws Exception {
+        addToRetryQueue = EventDispatcher.class.getDeclaredMethod("addToRetryQueue", Message.class);
+        addToRetryQueue.setAccessible(true);
+        retryQueue = EventDispatcher.class.getDeclaredField("retryQueue");
+        retryQueue.setAccessible(true);
+        subscribers = EventDispatcher.class.getDeclaredField("subscribers");
+        subscribers.setAccessible(true);
```

Comment:
I would make these methods and fields package-private instead, and then move the test class one level down into the `org.jenkinsci.plugins.ssegateway.sse` package so it can access them without reflection.